src/types/w3c-did.ts

```diff
@@ -0,0 +1,63 @@
+export interface Identity {
+  identified: true;
+  ethereumAddress: string;
+  smartContract: string;
+}
+
+export interface DIDDocument {
```

Comment:
I think you can use some of the types/methods from here https://github.com/decentralized-identity/did-common-typescript/blob/master/lib/IDidDocument.ts

Reply:
I actually pulled direct from the did-resolver here: https://github.com/decentralized-identity/did-resolver/blob/02bdaf1687151bb934b10093042e576ed54b229c/src/resolver.ts#L15

Reply:
would it make sense to then install it as a dependency and use the exported types?

Reply:
That feels like a good solution.  Then also as types inevitably evolve we can just bump version of the package instead of having to replicate changes manually... I like it.